events/wc_class_events.txt

```diff
@@ -900,6 +900,110 @@ character_event = {
 	}
 }
 
+# Selection of tamed beast
+character_event = {
+	id = WCCLS.110
+	desc = EVTDESC_WCCLS_110
+	picture = GFX_evt_hunter_human
+	border = GFX_event_normal_frame_war
+
+	is_triggered_only = yes
+
+	# Tortoise
+	option = {
+		name = EVTOPTA_WCCLS_110
+		random_list = {
+			60 = {
+				modifier = {
+					factor = 2
+					has_5_6_physical_class_or_higher_trigger = yes
+				}
+				modifier = {
+					factor = 2
+					has_7_8_physical_class_or_higher_trigger = yes
+				}
+				modifier = {
+					factor = 2
+					has_9_10_physical_class_trigger = yes
+				}			
+				add_character_modifier = { name = beast_tortoise duration = -1 }
+			}
+			30 = { add_trait_wounded_effect = yes }
+			10 = { add_maimed_trait_effect = yes }
+		}
+	}
+	# Eagle
+	option = {
+		name = EVTOPTB_WCCLS_110
+		random_list = {
+			55 = {
+				modifier = {
+					factor = 2
+					has_5_6_physical_class_or_higher_trigger = yes
+				}
+				modifier = {
+					factor = 2
+					has_7_8_physical_class_or_higher_trigger = yes
+				}
+				modifier = {
+					factor = 2
+					has_9_10_physical_class_trigger = yes
+				}			
+				add_character_modifier = { name = beast_snake duration = -1 }
+			}
+			33 = { add_trait_wounded_effect = yes }
+			12 = { add_maimed_trait_effect = yes }
+		}
+	}
+
+	# Prowler
+	option = {
+		name = EVTOPTC_WCCLS_110
+		random_list = {
+			50 = {
+				modifier = {
+					factor = 2
+					has_5_6_physical_class_or_higher_trigger = yes
+				}
+				modifier = {
+					factor = 2
+					has_7_8_physical_class_or_higher_trigger = yes
+				}
+				modifier = {
+					factor = 2
+					has_9_10_physical_class_trigger = yes
+				}			
+				add_character_modifier = { name = beast_panther duration = -1 } #Success
+			}
+			35 = { add_trait_wounded_effect = yes }
+			15 = { add_maimed_trait_effect = yes }
+		}		
+	}
+	
+	# Bear
+	option = {
+		name = EVTOPTD_WCCLS_110
+		random_list = {
+			35 = {
+				modifier = {
+					factor = 2
+					has_5_6_physical_class_or_higher_trigger = yes
+				}
+				modifier = {
+					factor = 2
+					has_7_8_physical_class_or_higher_trigger = yes
+				}
+				modifier = {
+					factor = 2
+					has_9_10_physical_class_trigger = yes
+				}			
+				add_character_modifier = { name = beast_bear duration = -1 }
+			}
+			35 = { add_trait_wounded_effect = yes }
+			30 = { add_maimed_trait_effect = yes }
```

Comment:
Wow, giant fail chances for the weak modifier. Needs rebalancing.

Reply:
Rebalanced. check it